install.sh

```diff
@@ -84,7 +84,13 @@
     URL="https://github.com/transifex/cli/releases/download/$LATEST_VERSION/tx-$OS-$ARCH.tar.gz"
 
     echo -e "** Installing CLI from $URL\n"
-    curl -L "$URL" | tar xz --skip-old-files
+    tar --version | grep -q 'gnu'
+    if [ $? -eq 0 ]
```

Comment:
I'm pretty sure this will work too (and be slightly easier to read):

```suggestion
    if tar --version | grep -q 'gnu'
```